src/types/ScreenInfoUiaProviderBase.h

```diff
@@ -32,9 +33,7 @@ namespace Microsoft::Console::Types
     class Viewport;
 
     class ScreenInfoUiaProviderBase :
-        public IRawElementProviderSimple,
-        public IRawElementProviderFragment,
-        public ITextProvider
+        public WRL::RuntimeClass<WRL::RuntimeClassFlags<WRL::ClassicCom>, IRawElementProviderSimple, IRawElementProviderFragment, ITextProvider>

```

Comment:
I don't honestly know whether these objects are supposed to be "agile". If they are, this is fine. If they're not, you can add the `InhibitFtmBase` flag to the existing `ClassicCom` flag to suppress some additional generated machinery.